com.bardsoftware.constable/src/main/kotlin/com/bardsoftware/constable/Main.kt

```diff
@@ -0,0 +1,14 @@
+package com.bardsoftware.constable
+
+import java.io.File
+
+fun main() {
+    println("Enter path to sql file with database schema to load")
+    val path = readLine()!!
```

Comment:
Maybe replace it with the command line argument? 
There is a choice of nice libs for that:
* https://ajalt.github.io/clikt/
* https://github.com/xenomachina/kotlin-argparser
* https://github.com/Kotlin/kotlinx.cli
* http://jcommander.org/

Wrt !!, I understand it is a corner case when the result of `readLine` call becomes null, but anyway, the canonical Kotlin null-safe way is like this:
```
readLine()?.let { path ->
  // Here path is not null
  if (!File(path).exists())
....
}
```

or like this:

```
val path = readLine() ?: return
// Here path is not null. If readLine returns null we return as well.
```

Reply:
Done